Python/thread_pthread.h

```diff
@@ -302,6 +308,27 @@ PyThread_get_thread_ident(void)
     return (unsigned long) threadid;
 }
 
+unsigned long
+PyThread_get_thread_id(void)
+{
+    if (!initialized)
+        PyThread_init_thread();
+#ifdef __APPLE__
+    volatile uint64_t tid;
+    pthread_threadid_np(NULL, &tid);
```

Comment:
On Apple platforms, the `pthread_threadid_np()` function wasn’t introduced until Mac OS 10.5, after Mac OS primarily became 64-bit. I figured it was acceptable to have the 64-bit type here since Python 3 for Mac only builds 64-bit versions and therefore the only Mac OS versions capable of supporting this feature would have to be those that also support Python 3 and therefore be 64-bit.

Reply:
Ok, sounds fine then :-) Thanks for clearing that up.